src/mmw/apps/modeling/tasks.py

```diff
@@ -210,6 +244,18 @@ def _get_precip_value(model_input):
         return None
 
 
+def convert_result_areas(pixel_width, results):
+    """
```

Comment:
Actually now that I think about it, since a pixel count in the area of interest is always available, it is probably easy to infer the area of each pixel given the area of the polygon (computer by `turf-area`) or similar.